### PR TITLE
Fix building application image

### PR DESCRIPTION
**Pull request by shamilkhan**

### CODE REVIEW

File: .github/workflows/prod.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-upload-static.yaml@1.0.0
     with:
       runs-on: '["self-hosted", "cere-network-large-workers"]'
-      build_container: 'node:14-buster'
+      build_container: 'node:lts-buster'
```
Comment (khssnv) [inline .github/workflows/prod.yaml:12]: Variable versioning may introduce non-deterministic builds. Is it intentional or we may use something like`node:18-buster` here instead?

File: packages/apps-config/src/api/params/inflation.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 import type { ApiPromise } from '@polkadot/api';
 
-import { DOCK_POS_TESTNET_GENESIS, KUSAMA_GENESIS, NEATCOIN_GENESIS, NFTMART_GENESIS, POLKADOT_GENESIS } from '../constants';
+import { DOCK_POS_TESTNET_GENESIS, KUSAMA_GENESIS, NEATCOIN_GENESIS, NFTMART_GENESIS, POLKADOT_GENESIS } from '../constants.js';
```
Comment (khssnv) [inline packages/apps-config/src/api/params/inflation.ts:6]: Usually, it is done by webpack's `resolve.extensions` ([docs](https://webpack.js.org/configuration/resolve/#resolveextensions)). What option is preferred here?

File: .github/workflows/prod.yaml (hunk 1 above)
Comment (shamilkhan) [reply .github/workflows/prod.yaml:12]: Thank you for your comment!
1. LTS stands for "last stable version" and is usually updated once or twice a year.
2. The Polkadot-JS team has built all of their codebase and GitHub workflow for the LTS version. For example, all pipeline versions have the `lts` node version.

File: packages/apps-config/src/api/params/inflation.ts (hunk 2 above)
Comment (shamilkhan) [reply packages/apps-config/src/api/params/inflation.ts:6]: Thank you for your message! It appears that the issue you are experiencing is not related to Webpack, but rather with the TypeScript compiler. You received the error while running `yarn build`, which includes the command `tsc --build tsconfig.build.json`.

There was an error during the build: 
```sh
error TS2835: Relative import paths need explicit file extensions in EcmaScript
 imports when '--moduleResolution' is 'node16' or 'nodenext'.
```

There is a conversation on GitHub regarding this issue: https://github.com/microsoft/TypeScript/issues/49083.
The main solution suggested is to simply add the .js extension.

**Pull request merged**